types/node/timers.d.ts

```diff
@@ -40,18 +40,21 @@ declare module 'timers' {
         }
 
         function setTimeout<TArgs extends any[]>(callback: (...args: TArgs) => void, ms?: number, ...args: TArgs): NodeJS.Timeout;
+        function setTimeout(callback: (args: unknown) => void): NodeJS.Timeout;
```

Comment:
It might actually be better to use `void` rather than unknown.

Also please add a brief comment on why these rather weird overloads exist for future ref.

Reply:
I've updated the PR now with this feedback. I hope the comments make sense.